src/__tests__/configurePrim.test.tsx

```diff
@@ -141,29 +141,29 @@ describe('usePrim', () => {
     expect(result.current).toMatchObject({
       w: { sm: { width: 22 } },
       se: { w: { sm: { width: 22 } } },
-      x: { w: { sm: undefined } },
-      tablet: { w: { sm: undefined } },
+      x: { w: {} },
+      tablet: { w: {} },
     })
     setScreenSize(ScreenSize.tablet)
     expect(result.current).toMatchObject({
       w: { sm: { width: 22 } },
-      se: { w: { sm: undefined } },
-      x: { w: { sm: undefined } },
+      se: { w: {} },
+      x: { w: {} },
       tablet: { w: { sm: { width: 22 } } },
     })
     setScreenSize(ScreenSize.x)
     expect(result.current).toMatchObject({
       w: { sm: { width: 22 } },
-      se: { w: { sm: undefined } },
+      se: { w: {} },
       x: { w: { sm: { width: 22 } } },
-      tablet: { w: { sm: undefined } },
+      tablet: { w: {} },
     })
     setScreenSize(ScreenSize.se)
     expect(result.current).toMatchObject({
       w: { sm: { width: 22 } },
       se: { w: { sm: { width: 22 } } },
-      x: { w: { sm: undefined } },
-      tablet: { w: { sm: undefined } },
+      x: { w: {} },
+      tablet: { w: {} },
```

Comment:
I'm second-guessing these changes. It seems unlikely that existing code is indexing into one these _inactive_ objects, i.e. `prim.tablet.w.width`, but if they are, then this would result in crashes.

Reply:
At a minimum I should make sure the types reflect that these are optional. 🤔